modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/JournalPortlet.es.js

```diff
@@ -225,24 +223,26 @@ export default function _JournalPortlet({
 			availableLocalesInput.value = availableLocales;
 
 			if (autoSaveDraftEnabled && !redirectOnSave) {
-				Liferay.componentReady(`${namespace}dataEngineLayoutRenderer`)
-					.then((dataEngineLayoutRenderer) => {
-						const dataEngineLayoutRendererRef =
-							dataEngineLayoutRenderer?.reactComponentRef;
-
-						return dataEngineLayoutRendererRef.current.validate();
-					})
-					.then((validForm) => {
-						if (validForm) {
-							removeAlert();
-							submitAsyncForm(form, {redirectOnSave});
-						}
-						else {
-							Liferay.fire('ddmFormError', {
-								formWrapperId: formId,
-							});
-						}
-					});
+				if (showErrors) {
+					Liferay.componentReady(
+						`${namespace}dataEngineLayoutRenderer`
+					)
+						.then((dataEngineLayoutRenderer) => {
+							const dataEngineLayoutRendererRef =
+								dataEngineLayoutRenderer?.reactComponentRef;
+
+							return dataEngineLayoutRendererRef.current.evaluate();
```

Comment:
Just confirming (because I don't know exactly what it does): did you change `.validate()` to `.evaluate()` on purpose, isn't it?

Reply:
Yes I did, so it does not trigger the validation, but we know wether the form is valid or not